client/src/components/v3/status/WirelessModuleStatusCol.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Col, Card, Accordion, Button, Table } from 'react-bootstrap';
+import OnlineStatusPill, { isOnlineT } from 'components/common/OnlineStatusPill';
+
+export interface WirelessModuleStatusProps {
+    moduleName: string,
+    isOnline: isOnlineT,
+    sensorData: {},
+    batteryVoltage?: number,
+    mqttAddress?: string,
+};
+
+export default function WirelessModuleStatusCol({
+    moduleName,
+    isOnline,
+    sensorData,
+    batteryVoltage,
+    mqttAddress }: WirelessModuleStatusProps) {
+
+    return (
+        <Col md xl="12" className="my-2">
+            <span>
+                <b>{moduleName}</b> <OnlineStatusPill isOnline={isOnline} />
+            </span>
+
+            {/* Only show more information if the camera is online */}
+            {isOnline && <>
+                {/* MQTT address */}
+                <p style={{ fontSize: "0.75rem", color: "gray" }}>{mqttAddress}</p>
+
+                <Table striped bordered hover>
```

Comment:
Is there a reason you made this a table? I reckon you could just list the sensors on a line